public/.htaccess

```diff
@@ -277,4 +277,25 @@ RewriteRule ^ %{ENV:BASE}/index.php [L]
             application/xml font/opentype image/svg+xml image/x-icon \
             text/css text/html text/plain text/x-component text/xml text/javascript
     </IfModule>
-</IfModule>
\ No newline at end of file
+</IfModule>
+
+# ----------------------------------------------------------------------
+# | Security settings                                                  |
+# ----------------------------------------------------------------------
+
+<IfModule mod_headers.c>
+
+    Header set X-Content-Type-Options nosniff
+    Header set X-Frame-Options SAMEORIGIN
+    Header always unset x-powered-by
+
+    # Content Security Policy (see https://content-security-policy.com/ for more information and examples)
+    Header set Content-Security-Policy "default-src 'self';"
+
+    # HTST
+    Header set Strict-Transport-Security "max-age=2592000; includeSubDomains;" # 2592000 = 30 days
+    # Preload feature (see https://hstspreload.org/) -> https://hstspreload.org/#deployment-recommendations
+#    Header set Strict-Transport-Security "max-age=300; includeSubDomains;"
+#    Header set Strict-Transport-Security "max-age=63072000; includeSubDomains; preload"
```

Comment:
Any reason why those two headers are commented out?

Reply:
They recommend ramping up the max-age in stages, starting with a value of 300.
[https://hstspreload.org/#deployment-recommendations](https://hstspreload.org/#deployment-recommendations)

Reply:
I see, but then we should start with 300 (5 minutes) and not with 2592000 (1 month), right? So first 5 minutes, then 1 week (604800), after that comes 1 month and last but not least something like 2 years (63072000) as far as I understand.